connect/connect.go

```diff
@@ -1,24 +1,73 @@
 package connect
 
 import (
+	"errors"
 	"fmt"
+	"os"
+	"path/filepath"
 
 	"github.com/joshmedeski/sesh/config"
-	"github.com/joshmedeski/sesh/session"
 	"github.com/joshmedeski/sesh/tmux"
 	"github.com/joshmedeski/sesh/zoxide"
 )
 
-func Connect(choice string, alwaysSwitch bool, command string, config *config.Config) error {
-	session, err := session.Determine(choice, config)
+// Connect to an established tmux session if one with a name or path matching
```

Comment:
Please remove all the comments in the PR, I'd like the code to be self-documenting. If something needs to be explained it may need to be broken up into it's own additional function.

Reply:
Sure, I'll remove all the comments I added in this PR. 

FYI idiomatic Go should contain some amount of comments that are used to generate documentation, see https://go.dev/doc/comment

Reply:
Thanks for the heads up, for now let's remove them and I'll think about the standard I want to set for code comments.

Reply:
Yep, already removed all the comments from this PR.